pipelines/pipelines/obr-generic/build-branch.yaml

```diff
@@ -308,8 +308,8 @@ spec:
 # 
 # 
 # 
-  # Trigger Isolated when doing a Main Build
-  - name: trigger-isolated
+  # Trigger CLI when doing a Main Build
+  - name: trigger-cli
```

Comment:
I think you can change this section back to how it was originally, we still want obr-generic to trigger Isolated if a main build, and Eclipse if release.